Project.toml

```diff
@@ -6,8 +6,11 @@ version = "0.2.0"
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
 ElasticArrays = "fdbdab4c-e67f-52f5-8c3f-e7b388dad3d4"
+FastIceTools = "14532042-4700-46e0-8dec-55d517bc1ae0"
```

Comment:
I am unsure if these should go into main Project's toml or in the script folder only

Reply:
As per our discussion, this should go into the apps only and not the Project.toml